### ISSUE TITLE
[Bug] After converting Outlook .msg to EML, bullet lists have duplicate numbering HTML converted from RTF

### ISSUE THREAD

**Issue by nickrussler** [bbottema/simple-java-mail#530]
Converted with:
```
EmailConverter.outlookMsgToEML(new FileInputStream(msgFilePath))
```

![119992866-43b2f500-bfcb-11eb-9f9d-3b94a3c121c9](https://github.com/bbottema/simple-java-mail/assets/1096768/efba5fe7-4c3d-42c9-9fff-8833d1a2c878)

Attached the msg file and the eml file:
[test.msg.txt](https://github.com/user-attachments/files/15519424/test.msg.txt)
[text.eml.txt](https://github.com/user-attachments/files/15519423/text.eml.txt)

Originally reported in https://github.com/nickrussler/email-to-pdf-converter/issues/35

**Comment by bbottema** [comment]
This is an issue in the rtf-to-html sub library. There is currently no expectation of a fix.

**Comment by nickrussler** [comment]
Thanks for checking so fast and creating an issue in the right place

**Comment by bbottema** [comment]
Should be fixed in 8.11.2! Thanks again for your assistance.
